### ISSUE TITLE
Welcome email accessibility enhancement

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#385]
**Description of the issue:**
We currently lack alternative labels on elements in the welcome emails. The result of this is that in many places, elements are either ignored by a screenreader, or read out as non-meaningful content, i.e. the full name of an image

**Browser used:**
Voiceover on Chrome/Safari/Firefox

**OS used:**
OS X

**Device used:**
MacBook Pro

**Steps to reproduce:**

1. Enable voiceover on mac
2. Navigate to one of the welcome emails
3. Listen to the content spoken to the user

**Actual result:**
The content is not very accessible to the user - adding alternative text throughout the content would go a long way towards improving this

**Expected result:**
Alternative text should describe elements for users with visual impairments

**Comment by gregdorward** [comment]
@Roisi & @ngoepel , @Etiene has fixed these issues up in new versions of the 3 welcome emails. I can now navigate through the content with a screenreader much easier. I'm not sure what the process is with regards to putting these new templates into use, but can I leave it with you to close this ticket once we're using them please?

**Comment by gregdorward** [comment]
Closing this issue, as the new templates are available for use
